### ISSUE TITLE
Creating a Physical Domain File

### ISSUE THREAD

**Issue by jasonltorchinsky** [AlexandraLJones/MCBRaT3D#3]
I am attempting to create a physical domain file for use with the Monte Carlo driver. I have managed to compile all executables. My current process is:

1. Create a directory named `Data`, which will contain all output.
2. Run the executable made from the `cubic.f95` file in the `Domain-Files` directory.
3. Using information from [Issue #2 ](https://github.com/AlexandraLJones/MCBRaT3D/issues/2), create a namelist for use with the Mote Carlo driver.
4. Run the Monte Carlo driver.

However, when running the Monte Carlo driver, it produces the following error:
```
 Using value MonteCarloDriver.nml
 read_Common: ./Data/cubic_Absorbing.dom problem reading DENSITY, massConc, or Reff.
 History:
   read_Common: ./Data/cubic_Absorbing.dom problem reading DENSITY, massConc, or Reff.
```

This leads me to suspect one of two issues:

1. I notice that in the `write_Domain` subroutine of `OpticalProperties.f95`, there doesn't appear to be any call to write the density, mass concentration, or effective radius information to file. *Is there a different write_Domain subroutine that does indeed write this information?*
2. *Is there is another executable that I should run on the domain file that was generated by the executable associated with `cubic.f95` to generate the density, mass concentration, or effective radius information?* I hypothesized that it would be the executable generated by `PhysicalPropertiesToDomain.f95`, but this program doesn't seem to read a domain file.

Thank you for your time!

**Comment by AlexandraLJones** [comment]
Most of the source code in the Domain-Files directory was imported from my original fork of the I3RC, but I didn't make updates to most of it. I think the only programs that will produce domain files compatible with MCBRAT are homogBBDomain.f95 and inhomogBBDomain.f95. Unfortunately, I didn't save an input files for those programs. But I think this is what I used during the development phase when I was testing the model against analytical cases. Once I moved on to more complex cases I think I used the PhysicalPropertiestoDomainNew.f95 program in the Tools directory to create the physical domain file. The comments in that code look fairly comprehensible (thank you old me). So I think you could understand the inputs and outputs to this code.

TL;DR: use Tools/PhysicalPropertiestoDomainNew.f95 to create your physical domain file; I'll drop a sample domain file in the "run" directory.

**Comment by jasonltorchinsky** [comment]
Excellent! Thank you for the help. I was able to download and properly read the physical domain file that you provided 👍🏻
